types/window-or-global/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // Type definitions for window-or-global 1.0
 // Project: https://github.com/purposeindustries/window-or-global
 // Definitions by: vvakame <https://github.com/vvakame>
+//                 Jason Kwok <https://github.com/JasonHK>
```

Comment:
Arguably, you could add me here as well:
```suggestion
//                 Jason Kwok <https://github.com/JasonHK>
//                 ExE Boss <https://github.com/ExE-Boss>
```

Reply:
Updated